reports/api.py

```diff
@@ -110,6 +111,9 @@ class ChildSerializer(serializers.ModelSerializer):
         help_text="Array of ISO 639-3 (language) or ISO 639-5 (language family) "
         "alpha-3 codes. Value `__OTHER__` means any other language."  # noqa
     )
+    is_obsolete = serializers.SerializerMethodField(
+        help_text="Is all the child's guardian users instances marked as obsolete?"
```

Comment:
How about
"Are all the child's guardian user instances marked as obsolete?" or
"Are all the child's guardian users' instances marked as obsolete?"